src/lib/components/PlayersView.svelte

```diff
@@ -34,6 +58,8 @@
             class="bg-black text-white p-1 rounded-md px-1"
             on:click={() => onLoadPlayer(p)}>Load</button
           >
+        {:else if $isTrackedPlayerGM}
+          <div>Me</div>
```

Comment:
Good point. I will put "(you)" to keep it consistent with OBR's Player list. I will also remove the "Me" when it's currently selected. I think the yellow highlight should be enough to show it's currently selected and I think "In Sync" is a bit misleading since you're seeing your own characters.

Reply:
Done in b8c63cd.

![image](https://github.com/user-attachments/assets/4a5b4526-d40f-47df-9512-000cefb62660)